plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/impl/MapImpl.java

```diff
@@ -1407,36 +1431,35 @@ public void setRenderManagerInternalGen(RenderManager newRenderManagerInternal)
     }
 
     Adapter adapter = new AdapterImpl() {
-        /**
-         * @see org.eclipse.emf.common.notify.impl.AdapterImpl#notifyChanged(org.eclipse.emf.common.notify.Notification)
-         */
         @Override
         public void notifyChanged(Notification msg) {
             switch (msg.getFeatureID(RenderManager.class)) {
-            case RenderPackage.RENDER_MANAGER__VIEWPORT_MODEL_INTERNAL: {
+            case RenderPackage.RENDER_MANAGER__VIEWPORT_MODEL_INTERNAL:
```

Comment:
IMHO the code looks much better but since it is a generated file I assume:
* its reset after generation from EMF model again
* its worth to find a way how the right formatter is applied after code-generation

I'd like to avoid efford the might be useless ..

Reply:
Where can I find the information which class is generated? Is there any documentation I may read regarding EMF?